packages/cerebral-provider-forms/src/form.js

```diff
@@ -61,6 +61,9 @@ export default function computedForm (formValueTag) {
   return compute(
     formValueTag,
     (formValue) => {
+      if (typeof formValue !== 'object') {
```

Comment:
This would still fail on `null` actually. Have to check for that as well :) Possibly also `!Array.isArray(formValue)` ... not sure how an initial array value be handled, maybe it works? No test for it though